.github/workflows/macos.yml

```diff
@@ -28,7 +28,7 @@ env:
 
 jobs:
   build:
-    runs-on: macos-11
+    runs-on: macos-latest
```

Comment:
Is it a good idea to always use the latest? Won't this result in the tests suddenly breaking

Reply:
If our tests break by merely upgrading the OS don't we have bigger problems?

We also don't have anyone taking care of macOS who would possibly remember to increase the version here. So I think the case is pretty strong to just let Github decide for us.